deckhouse-controller/pkg/controller/module-controllers/release/controller.go

```diff
@@ -417,14 +424,36 @@ func (c *moduleReleaseReconciler) reconcilePendingRelease(ctx context.Context, m
 			return ctrl.Result{RequeueAfter: defaultCheckInterval}, nil
 		}
 	} else {
-		if e := c.updateModuleReleaseStatusMessage(ctx, mr, fmt.Sprintf("Update policy not set. Create a ModuleUpdatePolicy object and label the release '%s=<policy_name>'", UpdatePolicyLabel)); e != nil {
-			return ctrl.Result{Requeue: true}, e
+		// get all policies regardless of their labels
+		var policies = new(v1alpha1.ModuleUpdatePolicyList)
+		err = c.client.List(ctx, policies)
+		if err != nil {
+			return ctrl.Result{Requeue: true}, err
+		}
+		policy, err = c.getReleasePolicy(mr.GetModuleSource(), mr.GetModuleName(), policies.Items)
+		if err != nil {
+			if e := c.updateModuleReleaseStatusMessage(ctx, mr, "Update policy not set. Create a suitable ModuleUpdatePolicy object"); e != nil {
+				return ctrl.Result{Requeue: true}, e
+			}
+			return ctrl.Result{RequeueAfter: defaultCheckInterval}, nil
+		}
+		patch, _ := json.Marshal(map[string]any{
+			"metadata": map[string]any{
+				"labels": map[string]any{
+					UpdatePolicyLabel: policy.Name,
+				},
+			},
+		})
+		p := client.RawPatch(types.MergePatchType, patch)
+
+		err = c.client.Patch(ctx, mr, p)
```

Comment:
I think we have to patch Status here also. Otherwise we have something like:
```
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  labels:
    module: test
    modules.deckhouse.io/update-policy: test-alpha
    release-checksum: 4356368c17e0d4e13c94209d85557acb
    source: test
  name: test-v0.9.0
spec:
  ...
status:
  approved: false
  message: Update policy some-wrong-policy not found
  ...
```

Yes, it will be fixed after 15 minutes, but I guess we can fix it here

Reply:
DONE